github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-presubmits-0.54.yaml

```diff
@@ -0,0 +1,1940 @@
+  - always_run: true
+    branches:
+    - release-0.54
+    cluster: prow-workloads
+    context: pull-kubevirt-e2e-k8s-1.23-sig-compute-cgroupsv2
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 7h0m0s
+    labels:
+      preset-bazel-cache: "true"
+      preset-bazel-unnested: "true"
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-shared-images: "true"
+    max_concurrency: 11
+    name: pull-kubevirt-e2e-k8s-1.23-sig-compute-cgroupsv2-0.54
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/sh
+        - -c
+        - automation/test.sh
+        env:
+        - name: TARGET
+          value: k8s-1.23-sig-compute
+        - name: KUBEVIRT_CGROUPV2
+          value: "true"
+        image: quay.io/kubevirtci/bootstrap:v20220512-78048f1
+        name: ""
+        resources:
+          requests:
+            memory: 29Gi
+        securityContext:
+          privileged: true
+      nodeSelector:
+        type: bare-metal-external
+  - always_run: true
+    branches:
+    - release-0.54
+    cluster: prow-workloads
+    context: pull-kubevirt-e2e-k8s-1.23-sig-storage-cgroupsv2
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 7h0m0s
+    labels:
+      preset-bazel-cache: "true"
+      preset-bazel-unnested: "true"
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-shared-images: "true"
+    max_concurrency: 11
+    name: pull-kubevirt-e2e-k8s-1.23-sig-storage-cgroupsv2-0.54
+    optional: true
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/sh
+        - -c
+        - automation/test.sh
+        env:
+        - name: TARGET
+          value: k8s-1.23-sig-storage
+        - name: KUBEVIRT_CGROUPV2
+          value: "true"
+        image: quay.io/kubevirtci/bootstrap:v20220512-78048f1
+        name: ""
+        resources:
+          requests:
+            memory: 29Gi
+        securityContext:
+          privileged: true
+      nodeSelector:
+        type: bare-metal-external
+  - always_run: true
+    branches:
+    - release-0.54
+    cluster: prow-workloads
+    context: pull-kubevirt-e2e-k8s-1.23-sig-compute-nonroot
+    decorate: true
+    decoration_config:
+      grace_period: 5m0s
+      timeout: 7h0m0s
+    labels:
+      preset-bazel-cache: "true"
+      preset-bazel-unnested: "true"
+      preset-dind-enabled: "true"
+      preset-docker-mirror-proxy: "true"
+      preset-shared-images: "true"
+    max_concurrency: 11
+    name: pull-kubevirt-e2e-k8s-1.23-sig-compute-nonroot-0.54
+    spec:
+      containers:
+      - command:
+        - /usr/local/bin/runner.sh
+        - /bin/sh
+        - -c
+        - automation/test.sh
+        env:
+        - name: TARGET
+          value: k8s-1.23-sig-compute
+        - name: FEATURE_GATES
+          value: NonRoot
```

Comment:
I see that this used to be `NonRootExperimental`. Was the feature gate name changed?

Reply:
@iholder-redhat From what I see [here](https://github.com/kubevirt/kubevirt/blob/abc9d0d079da13f606efcef6512251e8e784457a/pkg/virt-config/feature-gates.go#L46) the string `NonRootExperimental` is deprecated in favor to `NonRoot`